_api-reference/nodes-apis/nodes-stats.md

```diff
@@ -1278,6 +1300,33 @@ admission_control.global_cpu_usage.transport.rejection_count.indexing | Integer
 admission_control.global_io_usage.transport.rejection_count.search | Integer | The total number of search rejections in the transport layer when the node IO usage limit was met. Any additional search requests are rejected until the system recovers. The CPU usage limit is configured in the `admission_control.search.io_usage.limit` setting (Linux only).
 admission_control.global_io_usage.transport.rejection_count.indexing | Integer | The total number of indexing rejections in the transport layer when the node IO usage limit was met. Any additional indexing requests are rejected until the system recovers. The IO usage limit is configured in the `admission_control.indexing.io_usage.limit` setting (Linux only).
 
+### `caches`
+
+Because this API supports the experimental [tiered caching feature]({{site.url}}{{site.baseurl}}/search-plugins/caching/tiered-cache/), its responses may change. If the tiered caching feature flag is not enabled, it will return 0 for all values.
+{: .warning}
+
+The `caches` object contains cache stats. Currently its only sub-metric is `request_cache`. The total values within each sub-metric are always returned regardless of the value of the query parameter `level`. 
+
+Field | Field type | Description
+:--- | :--- | :---
+request_cache | Object | Statistics for the request cache. 
+request_cache.size_in_bytes | Integer | The total size in bytes of the request cache. 
+request_cache.evictions | Integer | The total evictions from the request cache. 
+request_cache.hit_count | Integer | The total hit count for the request cache.
+request_cache.miss_count | Integer | The total miss count for the request cache.
+request_cache.item_count | Integer | The total number of items in the request cache.
+request_cache.store_name | String | The name of the store type used by the request cache. See [tiered cache]({{site.url}}{{site.baseurl}}/search-plugins/caching/tiered-cache/) for more information. 
+
+If the `level` query parameter is set to one of its valid values `indices`, `shard`, or `tier`, additional fields will be present in `caches.request_cache` which break down the values by these levels. 
+For example if `level=indices,tier`, the tiered cache is in use, and the node has indexes named `index0` and `index1`, we also have the same 5 metrics for each combination of level values: 
```

Comment:
What is the "we" in this context? In other words, what will have the same 5 metrics? 

```suggestion
For example if `level=indices,tier`, the tiered cache is in use, and the node has indexes named `index0` and `index1`, the `caches` object will contain same 5 metrics for each combination of level values: 
```